templates/named.conf.options.erb

```diff
@@ -127,3 +127,25 @@ statistics-channels {
     inet <%= @statistic_channel_ip %> port <%= @statistic_channel_port %><% if @statistic_channel_allow -%> allow { <%= @statistic_channel_allow.join('; ') %>; }<% end -%>;
 };
 <% end -%>
+
+<% if @query_log_enable -%>
+logging {
+
+        channel query_logging {
+                file "/var/cache/bind/named_querylog"
```

Comment:
Is there a variable you can use (`data_dir`) to remove some of the hard-coded pathiness?